### DISCUSSION TITLE
Have you ever heard of Desert Bus?

### DISCUSSION THREAD

**Discussion by CrystallineGeode** [Nrmot-GithubRepo/Project-CBT#47] [Ideas]
So, I had an idea while on vc with my friends, and was wondering if you could add this:

An item that is simply there showing that you got it. How did you get it? Simple:
Playing Desert Bus, and doing a round trip from Tucson to Las Vegas, then back to Tucson. You'd have to have your full attention on the game for all **_16 hours_**. Any slip-ups you do send you right back to the start, regardless of where you are on your trip.

I think this would be an excellent addition to Project CBT. What say you?
-Daisy

**Comment by Nrmot** [comment]
:trollface:
